internal/services/pam/testdata/fixtures/file_config_dir

```diff
@@ -0,0 +1 @@
+This should not be a file.
```

Comment:
TBH, you can rather have a helper creating the file.

Reply:
I figured, but it looked nice as well to have it explicitly listed in `testdata`, to show what we are testing even from looking at the fixtures, no?

Reply:
I was quite puzzled by it, but as you have "other config dir", this is a valid argument :)